manual/src/refman/lex.etex

```diff
@@ -1,5 +1,10 @@
 \section{s:lexical-conventions}{Lexical conventions}
 %HEVEA\cutname{lex.html}
+
+\subsubsection*{sss:lex:text-encoding}{Source file encoding}
+
+OCaml source files are expected to be valid UTF-8 encoded unicode text.
```

Comment:
_U_unicode

Reply:
Nit, but what do you think about using a more forceful wording:
```
OCaml source files must be valid UTF-8 encoded Unicode text.
The interpretation of source files which are not UTF-8 encoded is unspecified.
Such source files may be rejected in the future.
```

Reply:
There is a contradiction between the `must` in the first line and the following sentences. What do you think of:

> OCaml source files are expected to be valid UTF-8 encoded Unicode text.
The interpretation of source files which are not UTF-8 encoded is unspecified.
Such source files may be rejected in the future.

where the later sentences explain that the leniency in the first sentence is temporary.

Reply:
I would go with your first suggestion ("The interpretation of source files ... is unspecified"), which is short and to the point. Thanks!